tools/Dockerfile

```diff
@@ -27,4 +27,7 @@ RUN yes | pacman --sync --refresh \
         p7zip \
         python \
         python-netaddr \
-        terraform
+        terraform \
+        python-pip
```

Comment:
Nit: please sort by alphabetical order, and add new line at the end of the file

Reply:
updated.